gradle.properties

```diff
@@ -15,23 +15,32 @@ awsSDKVersion=1.11.762
 bucket4jVersion=4.4.1
```

Comment:
Do we use all the variables defined here? If not, please delete the unused ones.